src/_gettsim_tests/test_data/groupings/2023/skip_eltern_viele_kinder.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 info:
-  note: ''
+  note: Skipped because multiple fgs in hh.
```

Comment:
I think this statement is wrong here and for the next few tests, so not repeating. Sometimes there are multiple households indeed, but only one fg per household.